store/mockstore/mocktikv/mvcc_leveldb.go

```diff
@@ -722,12 +729,6 @@ func prewriteMutation(db *leveldb.DB, batch *leveldb.Batch, mutation *kvrpcpb.Mu
 		return errors.Trace(err)
 	}
 
-	// Check assertions.
-	if (ok && mutation.Assertion == kvrpcpb.Assertion_NotExist) ||
-		(!ok && mutation.Assertion == kvrpcpb.Assertion_Exist) {
```

Comment:
should this method still need check `ok && mutation.Assertion == kvrpcpb.Assertion_NotExist` ?

it seems checks inside `checkConflictValue` only cover `!ok` ?

Reply:
ok && mutation.Assertion == kvrpcpb.Assertion_NotExist

!ok && mutation.Assertion == kvrpcpb.Assertion_Exist

Now split into two different cases, the check is basically the same.